### DISCUSSION TITLE
documentation: For Monday (July 24) - updated theta-client section

### DISCUSSION THREAD

**Discussion by jcasman** [camera360developers/theta-client-guide#57] [Ideas]
Please finish work on #56

After that, for Monday, I would like to update the [React Native for theta-client 1.2 section](https://theta360developers.github.io/theta-client-guide/react-native-theta-client-1_2/ for _bitrate). It needs to be updated to 1.3. We do not need to keep old information. We want the whole section as much as up-to-date as possible. 

Look at the updates information here: https://github.com/ricohapi/theta-client/releases/tag/1.3.0

Look to test 

* getThetaModel() function 
* _bitrate (set to Fine, Normal, Economy)
  * Then test to see what happens to the filesize. I expect that you can make the filesize smaller but looking at the image, the quality does not diminish much. 

Here's where @codetricity tested different bitrate settings, but didn't use theta-client: https://community.theta360.guide/t/use-image-bitrate-to-optimize-clarity-and-file-size/9080

You can first test with curl or Insomnia, if that's easier. But after confirming that you can use the new features, please programm a new buttons and add them to the React Native Android app. If not possible, please let me know.

**Comment by AirTechWick** [comment]
Okay I'm on the _bitrate options now, seems like my information from "getThetaModel()" and "getOptions" are working on a real android device.
